test-resources/xml-bomb.txt

```diff
@@ -0,0 +1 @@
l1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; &lol1; </lolz>
```

Comment:
please change the file extension to align with the contents

Reply:
I left it as `.txt` as leaving it as `.xml` caused some unpleasant surprises when I tried to edit it (my editor froze up)

Reply:
oh... uh... yeah, that can happen. nvm then